src/ui/window.cpp

```diff
@@ -86,9 +86,9 @@ static Window::GLContextPtr init_gl(SDL_Window* window) {
   SDL_GL_SetAttribute(SDL_GL_DOUBLEBUFFER, 1);
   SDL_GL_SetAttribute(SDL_GL_ACCELERATED_VISUAL, 1);
   SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 4);
-  SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 5);
+  SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 1);
   // TODO: use SDL_GL_CONTEXT_PROFILE_CORE
-  SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_COMPATIBILITY);
+  SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
```

Comment:
This will not work. The code in renderer requires legacy opengl features: https://github.com/0xd34d10cc/shar/blob/3bb388380b60a18b5bf887fe9c12676adce3ebbf/src/ui/renderer.cpp#L135-L136

We have #142 to fix that.